politeiad/backend/gitbe/gitbe.go

```diff
@@ -2468,7 +2477,7 @@ func (g *gitBackEnd) setUnvettedStatus(token []byte, status backend.MDStatusT, m
 
 		// Update MD first
 		record.RecordMetadata.Status = backend.MDStatusVetted
-		record.RecordMetadata.Iteration += 1
+		record.RecordMetadata.Iteration++
```

Comment:
What's the point of changing all of these `+= 1`?

Reply:
![image](https://user-images.githubusercontent.com/10324528/87989638-af425100-cae2-11ea-9df9-96143a90e05b.png)

Apparently I have `go-lint` installed on my vs code and it complained about, It makes sense IMO, what do you think ?